packages/framework/src/Component/EntityLog/ChangeSet/Formatter/ScalarDataTypeFormatter.php

```diff
@@ -7,11 +7,14 @@
 class ScalarDataTypeFormatter
 {
     /**
-     * @param array $changes
+     * @param array{oldReadableValue: mixed, newReadableValue: mixed, oldValue: mixed, newValue: mixed} $changes
      * @return string
      */
     public function formatChanges(array $changes): string
     {
+        $changes['oldReadableValue'] = $changes['oldReadableValue'] ?: t('empty value');
+        $changes['newReadableValue'] = $changes['newReadableValue'] ?: t('empty value');
```

Comment:
I am not 100 % convinced about this change - the log entry might be confusing in some edge cases - if there is an actual string "empty value", and I change it to null, I get this message: :smile: 
![image](https://github.com/user-attachments/assets/dfde30a9-0606-4088-bf70-0bb3e387c27d)

Maybe the readable null representation should not be quoted, so it would say `...changed from "empty value" to empty value` but as I said, this is probably _unlikely-to-happen edge-case_ so I can live with it as it is :shrug: